src/app/page.tsx

```diff
@@ -1,101 +1,426 @@
+                },
+              }}
+            >
+              <Box
+                sx={{
+                  position: "absolute",
+                  top: uploadedFiles.length === 0 ? "50%" : "initial",
+                  left: uploadedFiles.length === 0 ? "50%" : "initial",
+                  transform:
+                    uploadedFiles.length === 0
+                      ? "translate(-50%, -50%)"
+                      : "none",
+                  width: "100%",
+                  height: "100%",
+                  display: "flex",
+                  flexDirection: "column",
+                  justifyContent:
+                    uploadedFiles.length === 0 ? "center" : "flex-start",
+                  alignItems:
+                    uploadedFiles.length === 0 ? "center" : "flex-start",
+                  textAlign: "center",
+                  maxWidth: "100%",
+                  maxHeight: "100%",
+                  objectFit: "contain",
+                  p: uploadedFiles.length !== 0 ? 2 : 0,
+                }}
+              >
+                <Typography
+                  variant="h5"
+                  color={theme.palette.text.primary}
+                  gutterBottom
+                >
+                  <b>
+                    {" "}
+                    {uploadedFiles.length > 0
+                      ? "Uploaded files" + " (" + uploadedFiles.length + ")"
+                      : "No uploaded files"}
+                  </b>
+                </Typography>
+                <Stack
+                  direction="column"
+                  spacing={2}
+                  sx={{ width: "100%", alignItems: "center" }}
+                >
+                  {uploadedFiles.map((file, index) => (
+                    <FileElement
+                      key={index}
+                      //handleRightClick={()=>console.log("handleRightClick")}
+                      //handleCloseContextMenu={()=>console.log("handleCloseContextMenu")}
+                      setDropZoneState={handleSetDropzoneState}
+                      //contextMenuAnchor={()=>console.log("contextMenuAnchor")}
+                      //setContextMenuAnchor={()=>console.log("setContextMenuAnchor")}
+                      fileName={file.getInfo().name}
+                      fileUrl={file.getInfo().path}
+                      //handleRename={()=>console.log("(newName) => handleRename(index, newName)")}
+                      handleDelete={() => {
+                        handleDelete(file.getInfo().path);
+                        setDropzoneState({ visible: false, image_url: null });
+                      }}
+                    />
+                  ))}
+                </Stack>
+              </Box>
+            </Box>
+          </Grid2>
+          <Grid2
+            size={{ xs: 10, md: 7 }}
+            sx={{ display: { xs: "none", md: "flex" } }}
+          ></Grid2>
+          <Grid2 size={{ xs: 10, md: 4 }}>
+            {uploadedFiles.length > 0 && (
```

Comment:
Why do we hide the submit button?

Reply:
You can also give it a disabled state. Might be better to hide it though. If we disable the button then we have to add a hint on the button to indicate why it is disabled.

Reply:
In forms, the submit button is usually disabled until form validation is passed. Technically this is not a form but still an indication that an action is required to go to the next step should be obvious from the get go. I'm okay with hiding it however.

Reply:
That's great. Can the tooltip be deactivated when the button is enabled?